README.md

```diff
@@ -17,7 +17,14 @@ Default shorcuts:
 
 First of all, you will need to have PostgreSQL installed and `psql` command in the `$PATH`.
 
-Then just clone this repo into your Neovim package folder:
+## Using Vimplug
+
+```
+Plug 'mzarnitsa/psql'
+```
+
+## Using git
```

Comment:
Thanks for the PR!

Can you please change this line to say `Using NeoVim's native package system`?